p2p/message_test.go

```diff
@@ -601,3 +603,53 @@ func TestDecodeTransactionMessageTwoExtrinsics(t *testing.T) {
 		t.Fatalf("Fail: got: %v expected %v", *decodedMessage, expected)
 	}
 }
+
+func TestDecodeConsensusMessage(t *testing.T) {
+	ID := binary.BigEndian.Uint32([]byte("BABE")) // 1111573061
+	encId := make([]byte, 4)
```

Comment:
can use `core/types.BabeEngineId` here instead

Reply:
Just updated the tests, please check again.